hack/virtc/README.md

```diff
@@ -41,9 +41,15 @@ _Download virtc_
 $ go get github.com/containers/virtcontainers
 ```
 
-_Build virtc_
+_Build and install pause binary_
 ```
 $ cd $GOPATH/src/github.com/containers/virtcontainers
+$ make
+$ make install
+```
+
+_Build virtc_
+```
```

Comment:
If we add virtc to our default built binaries, we would just call make and make install.